src/docs/overview/concepts.md

```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
     For more information about data risk classifications, see the [Information
     Security Risk Classification page][url_risk].
 
-## What's a cluster?
 
-A computing cluster is a federation of multiple compute nodes (independent
-computers), most commonly linked together through a high-performance
-interconnect network.
+## Investing into Sherlock
 
-What makes it a "super-computer" is the ability for a program to address
-resources (such as memory, CPU cores) located in different compute nodes,
-through the high-performance interconnect network.
-
-![overview](img/cluster_overview.png)
+For users who need more than casual access to a shared computing environment,
+Sherlock also offers Faculty members the possibility to invest into additional,
+dedicated computing resources.
 
-On a computing cluster, users typically connect to [login nodes][g_login],
-using a secure remote login protocol such as [SSH][g_ssh]. Unlike in
-traditional interactive environments, users then need to prepare [compute
-jobs][g_jobs] to submit to a [resource scheduler][g_scheduler]. Based on a set
-of rules and limits, the scheduler will then try to match the jobs' resource
-requirements with available resources such as [CPUs][g_cpu], memory or
-computing accelerators such as [GPUs][g_gpu]. It will then execute the user
-defined tasks on the selected resources, and generate output files in one of
-the different [storage locations][url_storage] available on the cluster, for
-the user to review and analyze.
+Unlike traditional clusters, Sherlock is a collaborative system where the
+majority of nodes are purchased and shared by the cluster users, known as
+*owners*. Choosing from a standard set of server configurations supported by
+SRCC staff (known as the Sherlock *catalog*), PIs can purchase their own
+servers to add to the cluster.
 
-[g_cpu]:        /docs/overview/glossary#cpu
-[g_gpu]:        /docs/overview/glossary#gpu
-[g_login]:      /docs/overview/glossary#login-nodes
-[g_ssh]:        /docs/overview/glossary#ssh
-[g_jobs]:       /docs/overview/glossary#job
-[g_scheduler]:  /docs/overview/glossary#scheduler
-[url_storage]:  /docs/storage/
+This model also allows Sherlock owners to benefit from the scale of the cluster
+by giving them access to more compute nodes than their individual purchase,
+which gives them much greater flexibility than owning a standalone cluster.
```

Comment:
Could we add a line about why that is possible? E.g., "owners share a group of nodes that, when not in use by the primary purchaser of the machine, can be used by other owners."

Reply:
It's detailed a little bit below with "The resource scheduler configuration works like this:", but I added a sentence to clarify.